data/aliases.ts

```diff
@@ -94,6 +96,7 @@ export const Aliases: {[alias: string]: string} = {
 	megacharizard: "Charizard-Mega-Y",
 	megacharizardx: "Charizard-Mega-X",
 	megacharizardy: "Charizard-Mega-Y",
+	megacruci: "Crucibelle-Mega",
```

Comment:
maybe a new section for cap mons mightbe worth it? i dont wanna conflate cap and real pokemon